src/actions/account.js

```diff
@@ -325,7 +325,16 @@ export const { addAccessKey, createAccountWithSeedPhrase, addAccessKeySeedPhrase
     CREATE_ACCOUNT_WITH_SEED_PHRASE: [
         async (accountId, recoveryKeyPair, fundingContract, fundingKey) => {
             await wallet.saveAccount(accountId, recoveryKeyPair);
-            await wallet.createNewAccount(accountId, fundingContract, fundingKey, recoveryKeyPair.publicKey)
+
+            try {
+                await wallet.createNewAccount(accountId, fundingContract, fundingKey, recoveryKeyPair.publicKey)
+            } catch (e) {
+                console.warn('RPC error: createNewAccount\n', e)
+                await new Promise((r) => setTimeout(r, 4000)) // wait 4s for chain
+                await wallet.getAccount(accountId).state()
+            }
+
+            // if not thrown getting new account state, account exists
```

Comment:
This means very different error will be thrown than actual problem with account creation, which isn't optimal.

I think ideally failure during account creation should go 2 directions depending on error:
- known fatal error (e.g. account with such name already exists) – allow user to change params and retry creation
- known non necessarily fatal error (e.g. retries) – keep user on the same page and just allow to hit Retry. Retry first checks if account already exists with correct keys. Also make sure to save the account info regardless, so that if user refreshes page and account was created it gets loaded ok.